de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/internal/SarosSession.java

```diff
@@ -1092,7 +1092,8 @@ public void disableQueuing(IReferencePoint referencePoint) {
 
     private SarosSession(final String id, IContainerContext context,
         IPreferenceStore localProperties, JID host,
-        IPreferenceStore hostProperties) {
+        IPreferenceStore hostProperties,
+        IReferencePointManager referencePointManager) {
 
         context.initComponent(this);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "initComponent"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWcmJq0JkEeD8QRkTSuL&open=AWcmJq0JkEeD8QRkTSuL)

[ISSUE_KEY]: AWcmJq0JkEeD8QRkTSuL